framework/utils.py

```diff
@@ -23,7 +23,7 @@
 from framework import users
 import settings
 
-from django.utils import feedgenerator
+from feedgenerator.django.utils import feedgenerator
```

Comment:
Thanks for figuring out how to make this still work without needing all of django.  However, the only code that uses this is no longer reachable.  In a follow-up PR we can remove it, and FeatureListXMLHandler and render_atom_feed.

Reply:
Created https://github.com/GoogleChrome/chromium-dashboard/issues/2366